piccolo_admin/example.py

```diff
@@ -176,8 +168,8 @@ async def booking_endpoint(request, data):
     Director,
     Movie,
     Studio,
-    User,
-    Sessions,
+    BaseUser,
+    SessionsBase,
```

Comment:
I think the reason this wasn't working is this:

<img width="545" alt="Screenshot 2022-06-18 at 08 56 26" src="https://user-images.githubusercontent.com/350976/174428610-8f281c9c-665c-4775-9a37-b7efa5b7c5af.png">

The `session_table` and `auth_table` arguments weren't being passed to `SessionsAuthBackend`. I think if that is changed, we can use the `User` and `Sessions` table as before.

Reply:
Unfortunately, this doesn't work because if the `Users` and `Sessions` table is in `example.py` I can't import them into `endpoints.py` (`ImportError` raised - circular import)